deploy/app/legacyclaims.tf

```diff
@@ -30,6 +30,31 @@ variable "legacy_block_index_table_region" {
 }
 
 
+variable "legacy_store_table_name" {
+  description = "The name of the legacy store DynamoDB table"
+  type = string
+  default = ""
+}
+
+# the store table is always deployed in us-west-2 for both prod and staging
+variable "legacy_store_table_region" {
+  description = "The region where the legacy store DynamoDB table is provisioned"
+  type = string
+  default = "us-west-2"
+}
+
+variable "legacy_blob_registry_table_name" {
+  description = "The name of the legacy blob registry table DynamoDB table"
+  type = string
+  default = ""
+}
+
+# the blob registry table is always deployed in us-west-2 for both prod and staging
```

Comment:
Ok that's definitely not right!

Reply:
ok, my bad, I was just monkey say monkey do copying some other lines.